tests/unit/testV2Shim.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-console */
+const { fork } = require('child_process');
+const path = require('path');
+const assert = require('assert');
+
+const checker = path.resolve(__dirname, '../utils/exportCheck');
+
+function execShim(version, env, callback) {
```

Comment:
Do you need `version` here? Can this not be decided using `process.env.ENABLE_UTAPI_V2`?